app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function name `copy_file` is not very descriptive. It would be better to name it something like `copy_txt_file`, since this function is specifically designed to copy text files.